### DISCUSSION TITLE
How do increase the memory allocated to WinHost.exe

### DISCUSSION THREAD

**Discussion by GamerGirlandCo** [tianocore/edk2#4829] [Q&A]
I'm trying to test a custom build of GRUB with EDK, but every time i try to run grubx64.efi, I get this error:

`Cannot allocate memory for memory map`.

I'm guessing this is because the emulator only has 128MB of RAM allocated. How do I fix this?

**Comment by ajfish** [comment]
> On Sep 13, 2023, at 9:35 AM, ☙◦ The Tablet ❀ GamerGirlandCo ◦❧ ***@***.***> wrote:
> 
> 
> I'm trying to test a custom build of GRUB with EDK, but every time i try to run grubx64.efi, I get this error:
> 
> Cannot allocate memory for memory map.
> 
> I'm guessing this is because the emulator only has 128MB of RAM allocated. How do I fix this?
> 

This project supports a lot of different VMs. You will likely have better luck getting an answer if you show how you launched. 

Thanks,

Andrew Fish
